spec/controllers/payments_controller_spec.rb

```diff
@@ -9,12 +9,14 @@
 
   describe 'POST#create' do
     let(:user) { create(:user) }
+    let(:young_user) { create(:user, :less_than_15_yo) }
     let(:headers) { Devise::JWT::TestHelpers.auth_headers({}, user) }
     let(:send_request) { post :create, params: payment_params.merge(user: user), format: :json }
+    let(:send_invalid_request) { post :create, params: payment_params.merge(user: young_user), format: :json }
     let(:payments_params) { ActionController::Parameters.new(payment_params) }
     let(:card_info) { { number: '4000 00100 0000 2230', exp_month: '2', exp_year: '2021', cvc: '314' } }
-    let(:payment_params) { { type: 'card', card: card_info } }
-    let(:user_payment_servise) do
+    let(:payment_params) { {   type: 'card', card: card_info } }
+    let(:user_payment_service) do
       payments_params.permit(:type, card: %i[number exp_month exp_year cvc]).merge(user: user)
     end
```

Comment:
it's still params
why called 'service'?

Reply:
hmm, why i changed it?
fixed